### PR TITLE
fix-issues

### PR DESCRIPTION
**Pull request by plopez26DPU**
closes #97 

fixed issues

### CODE REVIEW

**Comment by plopez26DPU** [comment]
@aaryan2134 , I just started new pull request, the other one was all goofed up

File: README.md [hunk 1]
```diff
@@ -7,43 +7,49 @@ List of resources and project samples for Engineering Students
...
   
 
 ## Introduction
 
 This is a list of resources for engineering students. This list is maintained by students for students. If you wish to contribute, please read the [contributing guidelines](CONTRIBUTING.md).
 If the PR is not according to the guidelines then it will not be accepted.
 
-## Student Offers <a name="student-programs"></a>
+## Student Offers 
+<a name="student-programs"></a>
```
Comment (aaryan2134) [inline README.md:43]: This is not doing anything as such. You can remove these from all sections

File: README.md [hunk 2]
```diff
@@ -7,43 +7,49 @@ List of resources and project samples for Engineering Students
...
+  - [Database Management System](#database-management-system)
+  - [MySQL](#mysql)
+  - [Java Programming](#java-programming)
+  - [Cyber Security Resources](#cyber-security-resources)
+    - [Must:](#must)
+    - [Free Resources:](#free-resources)
+    - [Youtube Playlists:](#youtube-playlists)
+  - [Shell Scripting](#shell-scripting)
+  - [App Development](#app-development)
+  - [DEV OPS](#dev-ops)
```
Comment (aaryan2134) [inline README.md:31]: indentation issue is still there for this section as compared to the rest

File: README.md [hunk 3]
```diff
@@ -7,43 +7,49 @@ List of resources and project samples for Engineering Students
...
-  - [Object Oriented Programming <a name="object-oriented-programming"></a>](#object-oriented-programming-)
-  - [Blockchain <a name="blockchain"></a>](#blockchain-)
+  - [Student Offers](#student-offers)
+  - [CS Resources](#cs-resources)
+  - [Computer Science Fundamentals](#computer-science-fundamentals)
+  - [Kotlin Programming](#kotlin-programming)
+  - [R Programming](#r-programming)
+  - [Operating-System](#operating-system)
+  - [Links to College Subjects](#links-to-college-subjects)
+  - [EE Resources](#ee-resources)
```
Comment (aaryan2134) [inline README.md:17]: Please add full names for these sections

File: README.md [hunk 4]
```diff
@@ -67,7 +73,55 @@ If the PR is not according to the guidelines then it will not be accepted.
 20. [Data Structures and Algorithms in Java](https://www.youtube.com/watch?v=6iCHf7OZn6c&list=PL6Zs6LgrJj3tDXv8a_elC6eT_4R5gfX4d)
 21. [Data Structures Easy to Advanced Course-FreeCodeCamp](https://www.youtube.com/watch?v=RBSGKlAvoiM)
 
-## EE Resources <a name="education"></a>
+## Computer Science Fundamentals
```
Comment (aaryan2134) [inline README.md:76]: This section is empty

**Review by aaryan2134 — CHANGES_REQUESTED** [review]

**Comment by aaryan2134** [comment]
https://github.com/aaryan2134/Engineering-Resources/pull/97
Please refer to this PR as well. Same issues are there

**Comment by aaryan2134** [comment]
Some things have been removed and moved here and there as well. Please look into this again

**Comment by plopez26DPU** [comment]
I think you're wanting something that markdown can natively simplify. My original pull request was the correct method of implementation. I think the use you're looking for is misunderstood. Can you explain the issue better?

**Pull request closed without merging**